flow.yml

```diff
@@ -24,13 +30,15 @@ executors:
       CUDA_VISIBLE_DEVICES: 0  # change this if you have multiple GPU
       XLA_PYTHON_CLIENT_ALLOCATOR: platform  # https://jax.readthedocs.io/en/latest/gpu_memory_allocation.html
     replicas: 1  # change this if you have larger VRAM
-    needs: [gateway]
+    needs: [gateway, clip_encoder]
   - name: rerank
-    uses: ReRank
-    uses_with:
-      clip_server: grpcs://demo-cas.jina.ai:2096
-    py_modules:
-      - executors/rerank/executor.py
+    uses: jinahub+docker://CLIPTorchEncoder/latest
```

Comment:
`uses` is not needed for external usage, right?

Reply:
but is good for documenting, so that if people wants to host their own. This is a story we must tell